x-pack/plugins/security_solution/public/management/pages/trusted_apps/state/items_page.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface PageInfo {
+  index: number;
+  size: number;
+}
+
+export interface ItemsPage<T> {
+  items: T[];
+  pageInfo: PageInfo;
+  totalItemsCount: number;
+}
+
+export const pageInfosEqual = (pageInfo1: PageInfo, pageInfo2: PageInfo) =>
```

Comment:
I moved it to selectors. From the other side I kinda fail to see how it can be real selector based on state, the real selector is needRefreshOfListData. The function above is just simply functional decomposition to make code a bit more readable. It's just a function to compare two objects equality.